recipes/json_logging/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "json_logging" %}
```

Comment:
Do we want the package name to match the name on `pip` (dash instead of underscore)?

Reply:
Nice catch. `json-logging` is also the name used [in the source repo](https://github.com/bobbui/json-logging-python/blob/403d8221795a6808f96be9d24a9d97250fcd293e/setup.py#L14).

Pushed a fix, thanks.